src/client/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -519,8 +519,8 @@ public static IAuthorityValidator CreateAuthorityValidator(AuthorityInfo authori
                         }
```

Comment:
I think the better fix is to change line 516 from `IsCommonOrganizationsOrConsumersTenant` to `IsCommonOrOrganizationsTenant`. Smth like

```csharp
if (!requestAuthority.IsCommonOrOrganizations())
{
    return requestAuthority;        
}

if (requestAuthority.IsConsumers())
{
    // let consumers be there.
}
```